meetings/2025-03-19.md

```diff
@@ -0,0 +1,109 @@
+* Chat: https://matrix.to/#/#solid_specification:gitter.im
+* Repository: https://github.com/solid/specification
+* Status: Agenda
+
+
+## Present
+* [Michiel de Jong](https://michielbdejong.com)
+* [elf Pavlik](https://elf-pavlik.hackers4peace.net)
+* Jesse Wright
+* Hadrian Zbarcea
+* [TallTed // Ted Thibodeau](https://github.com/TallTed/) (he/him) (OpenLink Software)
+* Rui Zhao
+
+
+## Announcements
+
+### Meeting Guidelines
+* [W3C Solid Community Group Calendar](https://www.w3.org/groups/cg/solid/calendar).
+* [W3C Solid Community Group Meeting Guidelines](https://github.com/w3c-cg/solid/blob/main/meetings/README.md).
+* No audio or video recording, or automated transcripts without consent. Meetings are transcribed and made public. If consent is withheld by anyone, recording/retention must not occur.
+* Join queue to talk.
+* Topics can be proposed at the bottom of the agenda to be discussed as time allows. Make it known if a topic is urgent or cannot be postponed.
+
+### Participation and Code of Conduct
+* [Join the W3C Solid Community Group](https://www.w3.org/community/solid/join), [W3C Account Request](http://www.w3.org/accounts/request), [W3C Community Contributor License Agreement](https://www.w3.org/community/about/agreements/cla/).
+* [Solid Code of Conduct](https://github.com/solid/process/blob/main/code-of-conduct.md), [Positive Work Environment at W3C: Code of Ethics and Professional Conduct](https://www.w3.org/Consortium/cepc/)
+* Operating principle for effective participation is to allow access across disabilities, across country borders, and across time. Feedback on tooling and meeting timing is welcome.
+* If this is your first time, welcome! please introduce yourself.
+
+
+### Scribes
+* Michiel
+
+## Topics
+
+### Moving AddressBook from vcard to foaf
+
+* MdJ: See https://github.com/solid/contacts/issues/8Md
+* MdJ: ...
+* MdJ: VCard is controlled by IETF. If we change W3C ontology it will result in a fork.
+  ... https://lists.w3.org/Archives/Public/semantic-web/2025Mar/0041.html
+  ... This topic pops up in discussions since 1999
+* JW: I'm looking at VCard as RFC which was published in 2011. We could extend vocabulary which is a common practice. We can have `solid-contacts` namespace and define additional terms there.
+* MdJ: This is close to Dan's proposal of moving AddressBook to FOAF. Address books are not specific to Solid. FOAF has similar terms to VCard. 
+* JW: FOAF already has alternatives to Vcard...
+  ... I see Tim's disagreement to use FOAF
+* MdJ: Instead of migrating it to FOAF we
+* eP: What is a problem with reusing SIOC AddressBook?
+* MdJ: We have code that uses a term. What went wrong is squating vcard ontolgy. The mechanism that address book uses is different from what SIOC is using. We just want to publis ontology which is used by existing implementations.
+* ... databases have migrations, we shouldn't instead of just rename it have to use a different mechanism.
```

Comment:
I can't decipher this sentence/fragment
```suggestion
* ... databases have migrations, we shouldn't instead of just rename it have to use a different mechanism.
```